### ISSUE TITLE
Some ideas for developing Mask Language Modeling, Mask Image Modeling and Word-Patch Alignment for LayoutLMv3

### ISSUE THREAD

**Issue by 14H034160212** [microsoft/unilm#1076]
Hi, for anyone who interested in the implementation of LayoutLMv3. Transfomers have updated the code for mask image modeling and the code is based on DEIT. You can inherit the code to implement the Mask Image Modeling for LayoutLMv3 and also you can also inherit the code from RoBERTa to implement the mask language modeling. For the word-patch alignment, I am still in progress. Free feel to have any discussion.
Here are the links:
[RoBERTa mask language modeling](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling)

[DEIT mask image modeling example](https://github.com/huggingface/transformers/tree/main/examples/pytorch/image-pretraining#using-datasets-from--datasets)

[More ideas for developing word patch alignment](https://github.com/NielsRogge/Transformers-Tutorials/issues/152)

**Comment by suresh1505** [comment]
I am using LayoutLMv3 object detection but not able to get input_ids, bbox and attention_mask only getting imges. Can you help?

**Comment by van51** [comment]
Hi! Did you make any progress with the pre-training code? Any chance you could share something with us? :)
